buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -32,15 +32,16 @@ object Versions {
     const val SpringMockk = "3.0.1"
     const val Swagger = "1.6.2"
     const val AssetModel = "0.1.+"
-    const val P8eScope = "0.4.9"
+    const val P8eScope = "0.5.+"
     const val ProvenanceHdWallet = "0.1.15"
     const val ProvenanceClient = "1.1.1"
     const val Unirest = "3.13.6"
     const val KeyAccessLib = "0.2.+"
-    const val LoanPackage = "0.1.8"
+    const val LoanPackage = "0.1.10"
```

Comment:
Won't the publishing workflow fail if using the local "+1" version ahead of [what is available](https://github.com/provenance-io/loan-package-contracts/releases)? 
```suggestion
    const val LoanPackage = "0.1.9"
```